Automattic/wp-calypso
# Offer Jetpack Complete, CTA button component

Per 1203759331596262-as-1203791007934538/f this PR adds a new button component to the Offer Complete card

It adds a call to action button that will redirect to the cart with Complete added.  The other sends the user to their site's wp-admin page

![Screenshot 2023-02-15 at 12 04 24 PM](https://user-images.githubusercontent.com/12895386/219100777-4a11cd27-9733-475a-8fce-a01d1613a4a7.png)

I also added logic for generic redirects if there is no available site slug.  I'm not sure if that will ever be possible, but it seems a good idea to include it. 

Tracks events are not yet integrated.  I figured I would do that in another PR to sort of keep the work compartmentalized in my mind

## Testing Instructions

Have a hand-dandy JN site available that's connected to Jetpack, but has no upgrades.

Head over to the calypso.live link below (or roll your own with yarn-start)

Append /jetpack/connect/plans/complete/[site] to the base URL

You should see the two buttons on the card.  Click each one and verify the get complete drops you in the cart with the complete plan already selected.  The other one should just send you over to wp-admin.

## Pre-merge Checklist




---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203791007934538
